components/automate-ui/src/app/page-components/user-table/user-table.component.html

```diff
@@ -1,42 +1,47 @@
 <app-authorized [allOf]="getPermissionsPath" [overrideVisible]="overridePermissionsCheck">
+  <ng-container *ngIf="(users$ | async)?.length === 0">
+    <div class="team-users-empty-state">
+      <p>Add some users to get started!</p>
```

Comment:
tiny issue with this where the empty state message will flash again very briefly after adding some users to the team. rather than fixing that in this PR, we plan to pull in this issue as a follow-up https://github.com/chef/automate/issues/1006 which should make the problem go away